ironfish/src/account/accountsdb.ts

```diff
@@ -70,7 +70,7 @@ export class AccountsDB {
     value: {
       transaction: Buffer
       blockHash: string | null
-      submittedSequence: bigint | null
+      submittedSequence: number | null
```

Comment:
I think this might be a breaking change -- We serialize the value with `JsonEncoding`, which uses `IJSON` and appends an "n" to the end of bigints. You could probably test it by mining some blocks on the current version, checking your balance to make sure the wallet picked up the miners fee transactions, then switching to this branch and checking your balance.

I think it's okay to do a breaking change into staging since we already have breaking changes there anyway, we'll just want to check if that's the case, and tell people to reset their accounts database.

Reply:
Oh, I gave bad repro steps for this, it's actually the pending transactions that have this field set to a number/bigint. These are better steps:
1. Start two clean nodes, mining on one of them
2. Stop mining, then use `accounts:pay` to send money from the mining node to the other node (so that the transaction doesn't end up on a block)
3. Stop both nodes, change to this branch, then start them again. Error should display:
```
(node:65298) UnhandledPromiseRejectionWarning: TypeError: Cannot mix BigInt and other types, use explicit conversions
    at Accounts.rebroadcastTransactions (/Users/derek/repos/ironfish/ironfish/build/src/account/accounts.js:516:30)
```